### PR TITLE
feat!: set nodejs target for air-beautify

### PR DESCRIPTION
**Pull request by shamsartem**
BREAKING CHANGE: this package is now intended for use in nodejs

### CODE REVIEW

**Review by nahsi — APPROVED** [review]

File: tools/wasm/air-beautify-wasm/src/lib.rs [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021 Fluence Labs Limited
+ * Copyright 2023 Fluence Labs Limited
```
Comment (mikevoronov) [inline tools/wasm/air-beautify-wasm/src/lib.rs:2]: In AquaVM/Marine year in copyright, reflect the year when this file was first created (for historical reasons).

**Review by mikevoronov — APPROVED** [review]
utACK

File: tools/wasm/air-beautify-wasm/src/lib.rs (hunk 1 above)
Comment (shamsartem) [reply tools/wasm/air-beautify-wasm/src/lib.rs:2]: this change is needed to trigger CHANGELOG generation. You can always just use git to learn when file was created. I can revert this and change something else if you think it's important

Comment (monoid) [reply tools/wasm/air-beautify-wasm/src/lib.rs:2]: The correct value is 2022 :)

Comment (mikevoronov) [reply tools/wasm/air-beautify-wasm/src/lib.rs:2]: Looks so, ok, let's keep 2022

Comment (shamsartem) [inline tools/wasm/air-beautify-wasm/src/lib.rs:2]:
```suggestion
 * Copyright 2022 Fluence Labs Limited
```

**Pull request merged**